### DISCUSSION TITLE
Homebrew

### DISCUSSION THREAD

**Discussion by Emyjakarta** [community/community#129305] [Programming Help]
### Select Topic Area

Question

### Body

I am finding it difficult to install homebrew on my linux terminal.

emyjakarta@DESKTOP-55BBG0K:~$ curl -fsSL -o install.sh https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh
emyjakarta@DESKTOP-55BBG0K:~$ less install.sh
emyjakarta@DESKTOP-55BBG0K:~$ /bin/bash install.sh
==> Checking for `sudo` access (which may request your password)...
==> This script will install:
/home/linuxbrew/.linuxbrew/bin/brew
/home/linuxbrew/.linuxbrew/share/doc/homebrew
/home/linuxbrew/.linuxbrew/share/man/man1/brew.1
/home/linuxbrew/.linuxbrew/share/zsh/site-functions/_brew
/home/linuxbrew/.linuxbrew/etc/bash_completion.d/brew
/home/linuxbrew/.linuxbrew/Homebrew

Press RETURN/ENTER to continue or any other key to abort:
==> /usr/bin/sudo /bin/mkdir -p /home/linuxbrew/.linuxbrew/Homebrew
==> /usr/bin/sudo /bin/chown -R emyjakarta:emyjakarta /home/linuxbrew/.linuxbrew/Homebrew
==> Downloading and installing Homebrew...
remote: Enumerating objects: 271768, done.
remote: Counting objects: 100% (1424/1424), done.
remote: Compressing objects: 100% (770/770), done.
error: 2470 bytes of body are still expected1 MiB | 211.00 KiB/s
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: fetch-pack: invalid index-pack output
Failed during: /usr/bin/git fetch --force origin
emyjakarta@DESKTOP-55BBG0K:~$ rm -rf /home/linuxbrew/.linuxbrew/Homebrew
emyjakarta@DESKTOP-55BBG0K:~$ sudo /bin/bash install.sh
==> Checking for `sudo` access (which may request your password)...
Don't run this as root!
emyjakarta@DESKTOP-55BBG0K:~$ /bin/bash install.sh

**Comment by ebndev** [comment]
Hey @Emyjakarta, thanks for being a part of the GitHub Community!

Are you still having this issue?

**Reply by Emyjakarta** [reply]
Thanks @ebndev I was able to rectify the problem.

**Reply by ebndev** [reply]
Nice! I'm going to close this Discussion then 👍
